content/en/docs/collector/internal-telemetry.md

```diff
@@ -34,16 +36,21 @@ service:
       address: '0.0.0.0:8888'
 ```
 
-You can enhance the metrics telemetry level using the `level` field. The
-following is a list of all possible values and their explanations.
+You can adjust the verbosity of the Collector metrics output by setting the
+`level` field to one of the following values:
 
 - `none` indicates that no telemetry data should be collected.
-- `basic` is the recommended value and covers the basics of the service
-  telemetry.
-- `normal` adds other indicators on top of basic.
+- `basic` covers the basics of the service telemetry.
+- `normal` is the default level and adds other indicators on top of basic.
 - `detailed` adds dimensions and views to the previous levels.
 
-For example:
+Each verbosity level represents a threshold at which certain metrics are
+emitted. See the [lists of internal metrics](#lists-of-internal-metrics) for a
+breakdown of metrics by level.
+
+The default level for metrics output is `normal`. To see more or fewer metrics,
+you can change the default by modifying the level in the config
+`service::telemetry::metrics`. For example:
```

Comment:
```suggestion
The default level for metrics output is `normal`. To use another level, set `service.telemetry.metrics.level`:
```

Also, is it necessary to also set `address`? If not, remove it from the example, or maybe drop the example entirely -- assuming that the suggested prose given above is considered enough.